README.md

```diff
@@ -2,32 +2,68 @@ Upstream status (`master` branch):
 [![OfflineIMAP build status on Travis-CI.org](https://travis-ci.org/OfflineIMAP/offlineimap.svg?branch=master)](https://travis-ci.org/OfflineIMAP/offlineimap)
 [![OfflineIMAP code coverage on Codecov.io](https://codecov.io/gh/OfflineIMAP/offlineimap/branch/master/graph/badge.svg)](https://codecov.io/gh/OfflineIMAP/offlineimap)
 [![Gitter chat](https://badges.gitter.im/OfflineIMAP/offlineimap.png)](https://gitter.im/OfflineIMAP/offlineimap)
+[![Say thanks to OfflineIMAP](https://img.shields.io/badge/Say%20Thanks-!-______.svg)](https://saythanks.io/to/OfflineIMAP)
 
 Upstream status (`next` branch):
 [![OfflineIMAP build status on Travis-CI.org](https://travis-ci.org/OfflineIMAP/offlineimap.svg?branch=next)](https://travis-ci.org/OfflineIMAP/offlineimap)
 
-[offlineimap]: http://github.com/OfflineIMAP/offlineimap
-[website]: http://www.offlineimap.org
-[wiki]: http://github.com/OfflineIMAP/offlineimap/wiki
-[blog]: http://www.offlineimap.org/posts.html
+[![Github All Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)](https://github.com/OfflineIMAP/offlineimap/graphs/traffic)
+[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
+[![GitHub contributors](https://img.shields.io/github/contributors/OfflineIMAP/offlineimap.svg)](https://github.com/OfflineIMAP/offlineimap/graphs/contributors)
+[![Snap status](https://build.snapcraft.io/badge/snapcrafters/offlineimap.svg)](https://build.snapcraft.io/user/snapcrafters/offlineimap)
 
-Links:
-* Official github code repository: [offlineimap]
-* Website: [website]
-* Wiki: [wiki]
-* Blog: [blog]
+[![Python versions supported](https://img.shields.io/pypi/pyversions/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![Package stability](https://img.shields.io/pypi/status/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![PyPI status](https://badge.fury.io/py/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![Kit format](https://img.shields.io/pypi/format/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![Implementation](https://img.shields.io/pypi/implementation/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
 
-# OfflineIMAP
+[![GitHub last commit](https://img.shields.io/github/last-commit/OfflineIMAP/offlineimap.svg)](https://github.com/OfflineIMAP/offlineimap/commits/master)
```

Comment:
Users don't care. Remove this line.

Reply:
Users don't care when the last commit was?  In my opinion, the date of the last commit is valuable because it shows recent development/merge activity in this upstream repo, which is proof the project is relatively active and living, which encourages contributors and donors/sponsors! Isn't this a good thing?

Reply:
Ok, I get your point.

Let's move out all the info, badges, etc you'd like for this purpose. Create a page for this on the website. This way you'll be much more free about what to show (and much more free on editions, BTW).

I do intend the README to be the README for our users and developers. Not more.

Reply:
I understand.  I, for one, as a user and developer, appreciate having these statistical/status/informational badges visible on the `README.md` because for most visitors/users/developers, the github repo functions as the most visited, de facto, spiritual home page of the project.
* The new links help the user get from the `README.md`, to every valuable resource relevant to the software, in one hop (one click). Install steps, mailing list, report issue, get data, visit pypi, visit homebrew/macports, made config file, latest releases, how to contribute, etc.
* These new badges, give a reminder of the sense of the vibrant activity, the value of the application, the extent to which it's appreciated in the world, and essential for so many users. In reminding us of the importance of the software, these badges help drive further maintenance and development contributions, which are essential for its existence. They are small, but they are so incredibly powerful, when showing out in the open, on the main `README.md`.
* I can understand desire to create a text-only `README` containing only readable text and readable `http` links, for users who are trying to read the `README` in a text-only shell terminal.

Reply:
There are available KPI (clones, stars), already. A README with too many badges is just confusing. It should not look like a list of capabilities and proprieties.

Select and keep the most important ones, please.